File: Practice-of-loops/05_prime/prime-1.py
# 5.	迴圈的練習-prime
# 輸入一正整數，找出所有小於或等於的質數。

def prime(n):
    P = []
    for i in range(2,n):
        prime = True
        for j in range(2, int(i**(0.5)+1)):
            if i%j == 0:
                prime = False
                break
        if prime is True:
            P.append(i)    
    return P

n = int(input("請輸入一正整數"))
print(prime(n))

File: Practice-of-loops/05_prime/prime-1.js
// 5.	迴圈的練習-prime
// 輸入一正整數，找出所有小於或等於的質數。


function prime(n) {
    const P = [];
    for (let i = 2; i < n; i++) {
        let Prime = true;
        for (let j = 2; j <= Math.floor(Math.sqrt(i)); j++) {
            if (i % j === 0) {Prime = false;break;}
        }
        if (Prime) {P.push(i);}
    }
    return P;
}

console.log(prime(20));
